scripts/check-expo-dependencies.js

```diff
@@ -0,0 +1,45 @@
+module.exports = async ({ context, github, fetch, dependencyNames }) => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const expoVersion = require('../package.json').dependencies.expo
+  const expoVersionSanitized = expoVersion
+    .replaceAll('~', '')
+    .replaceAll('^', '')
+
+  const majorExpoVersion = `${expoVersionSanitized.substring(
+    0,
+    expoVersionSanitized.indexOf('.')
+  )}.0.0`
+
+  const expoData = await fetch('https://api.expo.dev/v2/versions/latest')
+  const expoDataOutput = await expoData.json()
+
+  const { relatedPackages } = expoDataOutput.data.sdkVersions[majorExpoVersion]
+  const dependencyNamesList = dependencyNames.split(',')
+
+  const shouldClosePR = [
+    ...Object.keys(relatedPackages),
+    'react',
```

Comment:
Q: why do we need to add these manual entries? Doesn't expo include these in their "related packages"?

Reply:
React and react native packages are present in the API but with different names:

<img width="396" alt="Screenshot 2023-04-21 at 08 54 33" src="https://user-images.githubusercontent.com/17121337/233629230-684778e8-ed88-41ef-a79c-d6b1f2216c23.png">

I manually declared them because I saw that's how it's done in `expo-doctor`: https://github.com/expo/expo-cli/blob/main/packages/expo-doctor/src/utils/getRemoteVersionsForSdkAsync.ts#L8

Reply:
ok thanks. are we not missing one then? :)

Reply:
My bad 🙄  Thanks for noticing